.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
Is the cache still supported with sudo required? https://github.com/Diaoul/subliminal/pull/513/files#diff-354f30a63fb0907d4ad57269548329e3L29